src/test/test-models/good/compound-assign/divide_equals_print.stan

```diff
@@ -16,9 +16,9 @@ transformed data {
   int x = 10;
   real y = 20;
   print("in transformed data");
-  print("x: ", x)
+  print("x: ", x);
```

Comment:
There are a few tests that did not have all the semicolons set. Fixed those as stanc3 is strict on these.